aiida_quantumespresso/calculations/pw2gw.py

```diff
@@ -37,16 +37,10 @@ def define(cls, spec):
         spec.output('eps', valid_type=orm.ArrayData,
             help='The `eps` output node containing 5 arrays `energy`, `epsX`, `epsY`, `epsZ`, `epsTOT`')
 
-        spec.exit_code(302, 'ERROR_OUTPUT_STDOUT_MISSING',
-            message='The retrieved folder did not contain the required stdout output file.')
         spec.exit_code(305, 'ERROR_OUTPUT_FILES',
             message='The eps*.dat output files could not be read or parsed.')
-        spec.exit_code(310, 'ERROR_OUTPUT_STDOUT_READ',
-            message='The stdout output file could not be read.')
         spec.exit_code(311, 'ERROR_OUTPUT_STDOUT_PARSE',
```

Comment:
This should also be generic and can be moved to base class

Reply:
Actually, this currently isn't returned in any parser. Maybe I should add a try/except with a generic `Exception` around the `parsed_data, stdout_logs = self.parse_stdout(stdout, **kwargs)` call in `BaseParser._retrieve_parse_stdout`?